arex-instrumentation-api/src/main/java/io/arex/inst/extension/matcher/IgnoreClassloaderMatcher.java

```diff
@@ -1,12 +1,23 @@
 package io.arex.inst.extension.matcher;
 
+import io.arex.agent.bootstrap.util.CollectionUtil;
 import io.arex.agent.bootstrap.util.StringUtil;
+import java.util.List;
 import net.bytebuddy.matcher.ElementMatcher;
 
 public class IgnoreClassloaderMatcher extends ElementMatcher.Junction.AbstractBase<ClassLoader> {
 
     static final String BYTE_BUDDY_PREFIX = StringUtil.removeShadePrefix("net.bytebuddy.");
-    private final ElementMatcher<ClassLoader> matcher;
+
+    private static final List<String> IGNORED_CLASSLOADER_PREFIXES = CollectionUtil.newArrayList(
+        "sun.reflect.", "jdk.internal.reflect.", IgnoreClassloaderMatcher.BYTE_BUDDY_PREFIX);
+    private ElementMatcher<ClassLoader> matcher;
+
+    public IgnoreClassloaderMatcher(List<String> ignoreClassLoaderPrefixes) {
+        if (ignoreClassLoaderPrefixes != null) {
```

Comment:
Thank you very much for you PR !! i think it's already perfect.  
the only little thing is that there's no need judge: `if (ignoreClassLoaderPrefixes != null)` , because if null return `Collections.emptyList()` at `ConfigManager#setIgnoreXXX` method.

Reply:
done